### ISSUE TITLE
Various Player Actions

### ISSUE THREAD

**Issue by NoNutNed** [tiltedphoques/Mod-Compatibility#22]
# Mod information
Allows for emotes and poses for your character to do
#### Download link
https://www.nexusmods.com/skyrimspecialedition/mods/22960
#### Dependencies
none
# Compatibility
Mod works as expected, emotes are synced. 

# Additional Information

Was made for the nightlies , but luckily still carries over to Reborn

**Comment by MaverickMartyn** [comment]
Not sure if it is supposed to, but when using the emotes to play an instrument, no audio is playing for anyone.
Other audio seems fine, for example the sweeping animation.

**Comment by Pingu-Pox** [comment]
> Not sure if it is supposed to, but when using the emotes to play an instrument, no audio is playing for anyone. Other audio seems fine, for example the sweeping animation.

Doesn't seem to work in single player (non-STR) as well, so likely no audio was meant to play.

**Comment by Crowlikedream** [comment]
Yeah the play instrument is just an animation, no music. This mod works as intended. Provides a very good experience when playing together as it brings a slice of an MMO to playing with your family and friends.
